pyquil/api/_quantum_computer.py

```diff
@@ -906,6 +906,12 @@ def get_qc(
             timeout=timeout,
         )
 
+    if noisy is not None and noisy:
```

Comment:
is the `is not None` necessary?

Reply:
Didn't think so, but wasn't sure if there was some Python quirk that made it necessary.

I'll take it out and put it back with a comment if tests fail.